protos/client_batch_submit.proto

```diff
@@ -62,12 +62,15 @@ message ClientBatchSubmitRequest {
 //   * OK - everything with the request worked as expected
 //   * INTERNAL_ERROR - general error, such as protobuf failing to deserialize
 //   * INVALID_BATCH - the batch failed validation, likely due to a bad signature
+//   * QUEUE_FULL - the batch is unable to be queued for processing, due to
+//        a full processing queue.  The batch may be submitted again.
 message ClientBatchSubmitResponse {
     enum Status {
         STATUS_UNSET = 0;
         OK = 1;
         INTERNAL_ERROR = 2;
         INVALID_BATCH = 3;
+        QUEUE_FULL = 4;
```

Comment:
This compatibility change should be noted in the release notes for the next RC version.